File: java/src/main/java/ch/frostnova/example/shoppingcart/service/ExchangeRateService.java
package ch.frostnova.example.shoppingcart.service;

import ch.frostnova.example.shoppingcart.model.CurrencyAmount;

public interface ExchangeRateService {

    String BASE_CURRENCY = "CHF";

    CurrencyAmount convert(CurrencyAmount currencyAmount, String toCurrency);
}

File: kotlin/src/main/kotlin/ch/frostnova/example/shoppingcart/service/ExchangeRateService.kt
package ch.frostnova.example.shoppingcart.service

import ch.frostnova.example.shoppingcart.model.CurrencyAmount

interface ExchangeRateService {

    companion object {
        const val BASE_CURRENCY = "CHF"
    }

    fun convert(currencyAmount: CurrencyAmount, toCurrency: String): CurrencyAmount
}